README.md

```diff
@@ -1,13 +1,13 @@
 # Command Line Tools for Development with IBM Maximo Asset Mangement
 [![Build Status](https://travis.ibm.com/maximo-ohio/maximodev-cli.svg?token=yJyC5zQ7wEuSAyYtDC53&branch=master)](https://travis.ibm.com/maximo-ohio/maximodev-cli)
 
-The `maximodev-cli` is a set of command line tools for developing with Maximo Asset Management, which accelerates tasks, such as creating add-ons, MBOs, field validations, etc.  
+The `maximodev-cli` is a set of command line tools for developing with Maximo Asset Management that accelerates tasks, such as creating add-ons, MBOs, field validations, etc.  
 
-`maximodev-cli` requires that [NodeJS](https://nodejs.org/en/) **version 8+** be installed.  If you do not have NodeJS **version 8** installed, then install/upgrade it first.
+`maximodev-cli` requires that [NodeJS](https://nodejs.org/en/) **version 8+** be installed. If you do not have NodeJS **version 8** installed, then install or upgrade it first.
```

Comment:
line 6 (and 8 and 10 and basically this comment applies to all references)
need to put the in front of maximo-dev-cli and put that identifing noun after it